apps/tangle-website/src/components/leaderboard/FAQSection.tsx

```diff
@@ -0,0 +1,41 @@
+import { Typography } from '@webb-tools/webb-ui-components';
+
+import { FAQAccordion } from '..';
+
+const faqItems = [
+  {
+    question: 'What is the Tangle Incentivized Testnet?',
+    answer:
+      'Kasd eos sanctus sea sed voluptua dolores est accusam sed, et lorem labore est stet, rebum sed dolor rebum erat sed. At labore sea ipsum diam. Sanctus amet takimata elitr.',
```

Comment:
Do you have the content for this?

Reply:
@monaiuu can you help me with this, I don't think the current design has the answers in the FAQ section!

Reply:
cc @thomivy